vars/getManifestSHA.groovy

```diff
@@ -14,7 +15,7 @@ Map call(Map args = [:]) {
 
     def lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
     def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: manifestLock))
-    String shasRoot = inputManifest.getSHAsRoot("${JOB_NAME}", args.platform, args.architecture)
+    String shasRoot = inputManifest.getSHAsRoot(jobName, args.platform, args.architecture)
```

Comment:
This needs to be a new location that doesn't consider platform and architecture.